docs/api/app.md

```diff
@@ -143,61 +144,20 @@ app({
 })
 ```
 
-<!-- The 1993 videogame DOOM did not have jumping as a movement option. "Rip and Tear!" was one of the infamous quotes of the protagonist DoomGuy in the 1996 Doom comic "Knee Deep in the Dead". -->
-
-#### `dispatch:`
-
-A dispatch initializer that can create a [custom dispatch function](../architecture/dispatch.md#custom-dispatching) to use instead of the default dispatch.
-
----
-
-## Instrumentation
-
-`app()` returns the [dispatch](../architecture/dispatch.md) function your app uses. This can be handy if you want to [control your app externally](#usage-within-non-hyperapp-projects).
-
----
+<!-- The 1993 videogame DOOM did not have jumping as a movement option. -->
 
-## Examples
+## `dispatch:`
 
-### Regular Usage
+A [dispatch initializer](../architecture/dispatch.md#dispatch-initializer) that can create a [custom dispatch function](../architecture/dispatch.md#custom-dispatching) to use instead of the default dispatch. Mainly used for middleware and debugging.
```

Comment:
"Mainly used for middleware and debugging". I would change to "Allows tapping in to dispatches, for debugging, testing, telemetry et c."

Reply:
changed